statsdaemon.go

```diff
@@ -100,7 +100,12 @@ func (s *StatsDaemon) Run(listen_addr, admin_addr, graphite_addr string) {
 	s.Clock = clock.New()
 	s.graphiteQueue = make(chan []byte, 1000)
 	log.Printf("statsdaemon instance '%s' starting\n", s.instance)
-	output := &common.Output{s.Metrics, s.metricAmounts, s.valid_lines, s.Invalid_lines}
+	output := &common.Output{
+		Bucket:   s.Metrics,
+		Value:    s.metricAmounts,
+		Modifier: s.valid_lines,
+		Sampling: s.Invalid_lines,
+	}
```

Comment:
this is not right. common.Output does not have these fields. look at the definition in in common/output.go